src/avro_json_decoder.erl

```diff
@@ -31,6 +31,10 @@
 
 -include("erlavro.hrl").
 
+-ifdef(TEST).
```

Comment:
I think we should not. I make export all only for those modules, which really need it. That is mainly because of Idea didn't understand idfef macros and do not highligt unexported. Also quick export is not working there.